Plugins/Messages.lua

```diff
@@ -383,18 +422,53 @@ function BigWigsMessages:SetupFrames()
 	t:SetAllPoints(self.frames.leftbutton)
 	t:SetBlendMode("ADD")
 	self.frames.leftbutton:SetHighlightTexture(t)
-	self.frames.leftbuttontext = self.frames.leftbutton:CreateFontString(nil,"OVERLAY")
+	self.frames.leftbuttontext = self.frames.leftbutton:CreateFontString(nil, "OVERLAY")
 	self.frames.leftbuttontext:SetFontObject(GameFontHighlight)
 	self.frames.leftbuttontext:SetText(L["Test"])
 	self.frames.leftbuttontext:SetAllPoints(self.frames.leftbutton)
 
+	self.frames.middlebutton = CreateFrame("Button", nil, self.frames.anchor)
+	self.frames.middlebutton.owner = self
+	self.frames.middlebutton:SetWidth(40)
+	self.frames.middlebutton:SetHeight(25)
+	self.frames.middlebutton:SetPoint("BOTTOMLEFT", self.frames.anchor, "BOTTOMLEFT", 65, 10)
+	self.frames.middlebutton:SetScript("OnClick", function()
+		BigWigsOptions:OpenMenu(self.frames.middlebutton)
```

Comment:
Spent a while trying to get these to start at the appropriate level like Plugins -> Messages in FubarPlugin OnMenuRequest but couldn't get it to work, if you happen to know how to do that would be cool